packages/react-tinacms-inline/src/inline-field-image.tsx

```diff
@@ -0,0 +1,106 @@
+/**
+
+Copyright 2019 Forestry.io Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+import * as React from 'react'
+import { InlineField } from './inline-field'
+import { useCMS } from 'tinacms'
+import { useDropzone } from 'react-dropzone'
+import { InputFocusWrapper } from './styles'
+
+interface InlineImageProps {
+  name: string
+  path?: string
+  parse(filename: string): string
+  uploadDir(form: any): string
+  children?: any
+}
```

Comment:
removes `path` and set the type for `form`

```suggestion
import { useCMS, Form } from 'tinacms'
import { useDropzone } from 'react-dropzone'
import { InputFocusWrapper } from './styles'

interface InlineImageProps {
  name: string
  parse(filename: string): string
  uploadDir(form: Form): string
  children?: any
}
```